src/userModifySrc.js

```diff
@@ -0,0 +1,105 @@
+const bcrypt = require('bcrypt');
+const db = require('../db/db');
+
+// Updating user
```

Comment:
Same comment, please update it to jsdoc style comment for example
`/**
 * Create new project
 * @param {string} title Project title
 * @param {string} description Project description
 * @param {object} user User information
 * @returns {object} newProjectResults
 * @throws {object} errorCodeAndMsg
 */`